src/constants.h

```diff
@@ -34,8 +34,3 @@
  * Pubkey hash length
  */
 #define PUBKEY_HASH_LEN 20
-
-/**
- * Maximum number of outputs on a transaction.
- */
-#define MAX_NUM_TX_OUTPUTS 255
```

Comment:
Why did you remove this?

Reply:
Unused since we do not test this anymore. The limit is enforced when we read the length of outputs as a 1 byte unsigned integer.